composer.json

```diff
@@ -15,18 +15,15 @@
     }
   ],
   "require": {
-    "php": ">=7.2.0",
-    "ext-json": "*",
-    "guzzlehttp/guzzle": "^6.0 | ^7.0",
+    "php": "^7.3 || ^8.0",
     "psr/cache": "^1.0",
-    "spatie/enum": "^2.3"
-    
+    "league/oauth2-client": "^2.6",
```

Comment:
This is already a requirement in the zoho library however I notice you are requiring 2.6 instead of 2.0 so I'm assuming there is a reason for that and that this is intended

Reply:
Yes, it has some issues in tests due to a new method added that we use, so i enforce it